scripts.js

```diff
@@ -13,50 +14,43 @@ addCopy.classList.add("fas", "fa-copy", "copy", "icon", "push");
 addDelete.classList.add("fas", "fa-times", "delete", "icon", "edit");
 
 // focus curson in input on load
-window.onload = getInput.focus();
+window.onload = inputElement.focus();
```

Comment:
Actually, everything should be wrapped inside the `window.onload` callback. Think of this as: _"only execute this code when the whole window content has loaded"_

Eg:
```js
window.onload = () => {
  const inputElement = document.querySelector('input');
  inputElement.focus();
  ...
  // rest of your code
};